src/main/java/org/broadinstitute/hellbender/tools/copynumber/arguments/GermlineCallingArgumentCollection.java

```diff
@@ -0,0 +1,74 @@
+package org.broadinstitute.hellbender.tools.copynumber.arguments;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.tools.copynumber.GermlineCNVCaller;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class GermlineCallingArgumentCollection implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @Argument(
+            doc = "Prior probability of alt copy number with respect to contig baseline state in the reference copy number.",
+            fullName = "p-alt",
```

Comment:
Perhaps extract these to `public static final` in this class.

Reply:
Done.